manifests/y/YmSoft/Nalgaeset/10.4/YmSoft.Nalgaeset.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+PackageIdentifier: YmSoft.Nalgaeset
+PackageVersion: 10.4
+PackageLocale: en-US
+Publisher: YmSoft
+PublisherUrl: http://moogi.new21.org/en/
+Author: Hong Minhee
+PackageName: Nalgaeset Hangul Input System
```

```suggestion
PackageName: 날개셋 한글 입력기
```
shows up as `날개셋 한글 입력기` for en-US devices.